provisioner/components/network/dynamic_ifcfg.j2

```diff
@@ -0,0 +1,9 @@
+TYPE={{ interface_type }}
```

Comment:
Move all template files to a directory.

Reply:
> Move all template files to a directory.

Done added jinja2 folder and moved template files under that folder.